lib/tool_shed/galaxy_install/install_manager.py

```diff
@@ -352,7 +352,7 @@ def install_package( self, elem, tool_shed_repository, tool_dependencies=None, f
                                             # Process the next matching <actions> tag, or any defined <actions> tags that do not
                                             # contain platform dependent recipes.
                                             log.debug( 'Error downloading binary for tool dependency %s version %s: %s' %
-                                                       ( str( package_name ), str( package_version ), str( tool_dependency.error_message ) ) )
+                                                       ( str( package_name ), str( package_version ), util.smart_str( tool_dependency.error_message ) ) )
```

Comment:
I think `util.unicodify( tool_dependency.error_message )` would be a more appropriate fix.